apps/token/project.json

```
@@ -0,0 +1,71 @@
+{
+  "root": "apps/token",
+  "sourceRoot": "apps/token/src",
+  "projectType": "application",
+  "targets": {
+    "build": {
+      "executor": "@nrwl/web:webpack",
+      "outputs": ["{options.outputPath}"],
+      "defaultConfiguration": "production",
+      "options": {
+        "compiler": "babel",
+        "outputPath": "dist/apps/token",
+        "index": "apps/token/src/index.html",
+        "baseHref": "/",
+        "main": "apps/token/src/main.tsx",
+        "polyfills": "apps/token/src/polyfills.ts",
+        "tsConfig": "apps/token/tsconfig.app.json",
+        "assets": ["apps/token/src/favicon.ico", "apps/token/src/assets"],
+        "styles": ["apps/token/src/styles.scss"],
+        "scripts": [],
+        "webpackConfig": "@nrwl/react/plugins/webpack"
+      },
+      "configurations": {
+        "production": {
+          "fileReplacements": [
+            {
+              "replace": "apps/token/src/environments/environment.ts",
+              "with": "apps/token/src/environments/environment.prod.ts"
+            }
+          ],
+          "optimization": true,
+          "outputHashing": "all",
+          "sourceMap": false,
```

chore (blocking): Needed for sentry

```suggestion
          "sourceMap": true,
```